test/IISTestSite/Startup.cs

```diff
@@ -35,6 +35,7 @@ public void Configure(IApplicationBuilder app)
             app.Map("/CheckEnvironmentLongValueVariable", CheckEnvironmentLongValueVariable);
             app.Map("/CheckAppendedEnvironmentVariable", CheckAppendedEnvironmentVariable);
             app.Map("/CheckRemoveAuthEnvironmentVariable", CheckRemoveAuthEnvironmentVariable);
+            app.Map("/UpgradeFeatureDetection", UpgradeFeatureDetection);
```

Comment:
Why do you have both /UpgradeFeatureDetection and StartupUpgradeFeatureDetection?

Reply:
One is for inprocess and one is for out of process. I can merge them together if you would prefer.

Reply:
Yes please, they appear to do the same thing.

Reply:
I merged the test classes together, however I realized there are still issues with merging the test sites together between in-process and out of process. When we call dotnet publish, we provide web.config file with hostingModel=inprocess/outofprocess and the processPath/arguments. I thought I could add this to the apphostconfig file we provide when calling iisexpress, however I am running into issues there. I'll ask you tomorrow about this.